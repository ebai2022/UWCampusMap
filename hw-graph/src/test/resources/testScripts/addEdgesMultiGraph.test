# Test edge creation with multiple graphs

# Create graphs
CreateGraph A
CreateGraph B

# Add nodes
AddNode A e1
AddNode A e2
AddNode B e3
AddNode B e4

# Add edges
AddEdge A e1 e2 first
AddEdge B e3 e4 third
AddEdge B e3 e4 second

# Check the edges
ListChildren A e1
ListChildren A e2
ListChildren B e3
ListChildren B e4