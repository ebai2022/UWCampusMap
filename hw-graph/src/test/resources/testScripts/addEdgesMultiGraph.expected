# Test edge creation with multiple graphs

# Create graphs
created graph A
created graph B

# Add nodes
added node e1 to A
added node e2 to A
added node e3 to B
added node e4 to B

# Add edges
added edge first from e1 to e2 in A
added edge third from e3 to e4 in B
added edge second from e3 to e4 in B

# Check the edges
the children of e1 in A are: e2(first)
the children of e2 in A are:
the children of e3 in B are: e4(second) e4(third)
the children of e4 in B are: